-

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env zsh
```

Comment:
Did you intend to add this file in the PR?

Reply:
nope :)